Landing "MyBike"

vashivan opened the pull request:
DEMO LINK](https://vashivan.github.io/layout_landing-page/)

anastasiia-tilikina reviewed (changes requested):
Well done! But let's fix some issues though

1. Your pic overlaps another on hover. Consider either increasing scaling a bit or making pic smaller on hover `scale(0.8)` for instance
![image](https://github.com/mate-academy/layout_landing-page/assets/108538921/3e98b857-e2db-49db-a74f-0de3d7b7abe9)

2. You have some default spaces in texarea
![image](https://github.com/mate-academy/layout_landing-page/assets/108538921/bb474880-21be-4a98-9214-a037d363f22a)

3. The link `about` in the menu doesn't work

4. Paddings are not applied for the button, check the syntax
![image](https://github.com/mate-academy/layout_landing-page/assets/108538921/5732ac39-9124-4386-b9c6-d0a97e77a7b7)

5. Use the same value for transition duration. It's a good practice.
now your menu opens too slowly.

vashivan commented:
Thank you for comments, because I could see autofill trouble on my devices and through browsers. I think I coped with it, because I still cannot notice it......
<img width="1470" alt="Знімок екрана 2023-09-06 о 21 33 05" src="https://github.com/mate-academy/layout_landing-page/assets/123917348/f7e4cf8e-09cd-4397-ad2f-ece6fe401d33">
<img width="1470" alt="Знімок екрана 2023-09-06 о 21 28 03" src="https://github.com/mate-academy/layout_landing-page/assets/123917348/8b804e40-d488-4789-9bdb-549901bcb401">

anastasiia-tilikina reviewed (changes requested):
here is your issue with additional spaces in textarea. closing tag must be right after opening one. otherwise HTML considers it as spaces and puts in textarea
![image](https://github.com/mate-academy/layout_landing-page/assets/108538921/486e1020-e179-4293-8619-81a83dcd3bd4)

let's fix it)

also unfortunately I cannot notise autofill effects but you can read [here](https://css-tricks.com/almanac/selectors/a/autofill/) and [here](https://css-tricks.com/snippets/css/change-autocomplete-styles-webkit-browsers/) about it)

vashivan commented:
Thank you! I have fixed html and added property to my css for .input. Hope all is ok now:)
<img width="379" alt="Знімок екрана 2023-09-07 о 13 25 06" src="https://github.com/mate-academy/layout_landing-page/assets/123917348/ff4f0d35-53c8-4dd1-849b-5484fc541e01">
<img width="452" alt="Знімок екрана 2023-09-07 о 13 25 56" src="https://github.com/mate-academy/layout_landing-page/assets/123917348/6b52d444-af76-46bd-b5e0-1ae91e1b6b3e">

maxim2310 reviewed (approved):
Good work!!!
